epochX/cudacpp/CODEGEN/PLUGIN/CUDACPP_SA_OUTPUT/model_handling.py

```diff
@@ -1523,7 +1523,7 @@ def generate_subprocess_directory_end(self, **opt):
     def edit_coloramps(self, config_subproc_map):
         """Generate coloramps.h"""
 
-
+        misc.sprint(config_subproc_map)
```

Comment:
```suggestion
```
looks like I forget to remove one debug statement. So feel free to remove it (or we will clean those anyway one day in the future :-)

Reply:
Very good thanks, change committed